platform/mbed_lib.json

```diff
@@ -102,6 +102,22 @@
         "cthunk_count_max": {
             "help": "The maximum CThunk objects used at the same time. This must be greater than 0 and less 256",
             "value": 8
+        },
+        "crash-capture-enabled": {
+            "help": "Enables crash context capture when the system enters a fatal error/crash.",
+            "value": false
+        },
+        "reboot-crash-report-enabled": {
+            "help": "Enables crash context capture when the system enters a fatal error/crash.",
+            "value": false
+        },
+        "error-reboot-max": {
```

Comment:
It's not clear what 0 means here. Sounds as if it means don't reboot, but it's actually reboot once? 1 would mean reboot twice?

Reply:
That's correct, 0 means it still causes 1 reboot. May be its confusing as you see it, its also tricky to document that I guess. I'm going to change that to - 0 means no reboot, 1 means 1 reboot etc. Hope that helps.